contents/tutorials/performance-marketing.md

```diff
@@ -0,0 +1,105 @@
+---
+title: How to track performance marketing in PostHog
+sidebar: Docs
+showTitle: true
+author: ['ian-vanagas']
+date: 2022-12-02
+featuredImage: ../images/tutorials/banners/performance-marketing.png
+topics: ['funnels', 'insights']
+--- 
+
+Performance marketing is paying for ads, attention, and clicks to your site where you try to convert them into users and customers. Companies use channels like Google, Facebook, other social media, ad networks, and sponsorships to do this. 
+
+In this tutorial, we’ll cover how to track your performance marketing campaigns, channels, traffic, and conversion using PostHog's product analytics suite.
+
+## Tracking performance marketing with UTMs
+
+The first step in performance marketing is ensuring proper tracking. To do this, you must add Urchin Tracking Modules (UTM) to your links in each channel. These are pieces of information in your URL with details about where the user came from. They look like `https://posthog.com/tutorials?utm_source=google&utm_medium=social`.
+
+PostHog automatically captures UTMs via our JavaScript library (and snippet). The ones we support are: 
+- `utm_source`: traffic source, like Google or Facebook
+- `utm_medium`: traffic medium, like social or paid (ad)
+- `utm_campaign`: related campaign, like `brand_awareness` or `holiday_sale`
+- `utm_content`: content location, like a specific post, button, image
+- `utm_term`: search term, like “analytics” or “product analytics”
```

Comment:
```suggestion
- `utm_term`: search term, like "analytics" or "product analytics"
```

I'm guessing this is Notion thing, but I tend to remove curly quote marks to avoid any weird visual gremlins.